src/form/Calendar/Calendar.tsx

```diff
@@ -102,12 +103,12 @@ export const Calendar: FC<CalendarProps> = ({
     [value]
   );
   const rangeStart = useMemo(
-    () => value?.[0] ?? date ?? new Date(),
-    [date, value]
+    () => value?.[0] ?? parseISO('') ?? new Date(),
```

Comment:
??

Reply:
Previously, we were displaying the current date at the start. Now, at the beginning, no date is selected, allowing us to choose a range.

Reply:
I don't really understand - there's now some dead code as it will never reach `new Date()` but as it is, we can't really blindly change functionality like this as it will affect anyone currently using this component when they upgrade to the latest